assets/src/edit-story/app/history/useHistoryReducer.js

```diff
@@ -42,9 +42,35 @@ const reducer = (size) => (state, { type, payload }) => {
         );
 
         if (isReplay) {
+          const currentEntry = state.entries[state.offset];
```

Comment:
@barklund 
The logic is rewritten based on [Paul's comment](https://github.com/google/web-stories-wp/pull/1065#issuecomment-611904360).

Haven't written the tests yet since would be good to get some feedback first.

Ignoring the current page of the original entry seemed to be the cleanest way to achieve always landing on the page where the recent history action happened, this, however, will override the `replayState` if the originally attempted replay didn't have the "correct" current page saved.

Thoughts?